Formula/f/fonttools.rb

```diff
@@ -18,22 +18,52 @@ class Fonttools < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "5f5882ed5c35176b03b9de36e08e9bf30a634a7a79d6cfc4d7c31062538a2e8f"
   end
 
-  depends_on "python@3.12"
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.11" => [:build, :test]
+  depends_on "python@3.12" => [:build, :test]
```

Comment:
don't think we need `python@3.11` so we should keep the runtime dep for the CLI. otherwise, this requires users to manually installed python like https://github.com/Homebrew/homebrew-core/issues/151248

```suggestion
  depends_on "python@3.12"
```